spyder/api/plugin_registration/registry.py

```diff
@@ -170,6 +203,16 @@ def _instantiate_spyder5_plugin(
         else:
             self.internal_plugins |= {plugin_name}
 
+        if external:
+            if not running_under_pytest():
```

Comment:
I think this `if` is not necessary because we're registering the `spyder_boilerplate` plugin now and testing that is correctly the case.